commands/tokens.go

```diff
@@ -0,0 +1,334 @@
+package commands
+
+import (
+	"errors"
+	"fmt"
+	"strconv"
+	"time"
+
+	"github.com/MakeNowJust/heredoc"
+	"github.com/digitalocean/doctl"
+	"github.com/digitalocean/doctl/commands/displayers"
+	"github.com/digitalocean/doctl/do"
+	"github.com/digitalocean/godo"
+	"github.com/spf13/cobra"
+)
+
+// Tokens creates the tokens command
+func Tokens() *Command {
+	cmd := &Command{
+		Command: &cobra.Command{
+			Use:     "token",
+			Aliases: []string{"tokens"},
+			Short:   "[BETA] Display commands for managing DigitalOcean API tokens",
+			Long: heredoc.Docf(`
+			[BETA] Display commands for managing DigitalOcean API tokens.
+
+			Use the sub-commands of %s to create, manage, or revoke DigitalOcean API tokens.`,
+				"`doctl token`",
+			),
+			Hidden: true,
+		},
+	}
+
+	tokenDetails := heredoc.Docf(`
+	- The ID of the token
+	- The name of the token
+	- The scopes applied to the token
+	- The timestamp for when the token will expire
+	- The date that it was last used
+	- The timestamp for when it was created
+	`)
+
+	CmdBuilder(cmd, RunTokenGet, "get <id|name>",
+		"Retrieve information about a DigitalOcean API token",
+		heredoc.Docf(`Display the details for a specific DigitalOcean API tokens on your account. This includes:
+
+		%s`, tokenDetails),
+		Writer, aliasOpt("g"), displayerType(&displayers.Tokens{}))
+
+	CmdBuilder(cmd, RunTokenList, "list",
+		"List your DigitalOcean API tokens",
+		heredoc.Docf(`Lists the following details for DigitalOcean API tokens on your account:
+
+		%s`, tokenDetails),
+		Writer, aliasOpt("ls"), displayerType(&displayers.Tokens{}))
+
+	tokensCreateDesc := heredoc.Docf(`Create a new DigitalOcean API token with granular scopes.
+
+		You can find all of the scopes that are available to be applied to a tokens using the
+		%s sub-command. For example, to create a token that can be used to create and manage
+		Droplets, use:
+
+		    doctl token create droplet-service-token --scopes droplet:create,droplet:read,droplet:delete,droplet:operate
+
+		By default, tokens will expire in 30 days (2592000 seconds) by default. The maximum
```

Comment:
The second "by default" is redundant.
```suggestion
		By default, tokens will expire in 30 days (2592000 seconds). The maximum
```